include/zephyr/bluetooth/l2cap.h

```diff
@@ -290,6 +311,68 @@ struct bt_l2cap_le_chan {
  */
 #define BT_L2CAP_LE_CHAN(_ch) CONTAINER_OF(_ch, struct bt_l2cap_le_chan, chan)
 
+/** @brief Fixed L2CAP Channel structure. Should be defined using the
+ *  @ref BT_L2CAP_FIXED_CHANNEL_DEFINE macro.
+ */
+struct bt_l2cap_fixed_chan {
+	/** @brief Channel Identifier (CID)
+	 *
+	 *  @note Shall be in the range 0x0001 to 0x003F (Core 3.A.2.1 v6.0). The CIDs in this range
+	 *        are reserved by the stack, so the application needs to make sure that the CID is
+	 *        not already in use. Table 2.1 in Core 3.A.2.1 and @ref l2cap_cid shows the
+	 *        CIDs used by the stack.
+	 */
+	uint16_t cid;
+
+	/** @brief Channel accept callback
+	 *
+	 *  This callback needs to be provided by the application, and is invoked when a new
+	 *  connection has been established. If accepting the connection, the user is expected to
+	 *  allocate memory with suitable alignment for the type @ref bt_l2cap_chan for the channel,
+	 *  and update the channel reference @p chan to point to the allocated memory. The channel
+	 *  should be initialized by assigning the callbacks to the @ref bt_l2cap_chan_ops field
+	 *  as follows:
+	 *  @code
+	 *  static int accept(struct bt_conn *conn, struct bt_l2cap_chan **chan)
+	 *  {
+	 *      // Allocation of fixed_chan and definition of the ops are assumed done prior.
+	 *      *chan = &fixed_chan;
+	 *
+	 *      **chan = (struct bt_l2cap_chan){
+	 *          .ops = &ops,
+	 *      };
+	 *
+	 *      return 0;
+	 *  }
+	 *  @endcode
+	 *  The allocated context needs to be valid for the lifetime of the channel, i. e.
+	 *  freeing of the memory can be done in the @ref bt_l2cap_chan_ops.released callback.
+	 *
+	 *  @param conn The connection that has been established.
+	 *  @param chan L2CAP channel reference.
+	 *
+	 *  @return 0 to accept the channel connection, negative error code otherwise.
+	 */
+	int (*accept)(struct bt_conn *conn, struct bt_l2cap_chan **chan);
+};
+
+/**
+ *  @brief Register a fixed L2CAP channel.
+ *
+ *  Define and register a fixed L2CAP channel of type @ref bt_l2cap_fixed_chan.
+ *
+ *  @note Fixed L2CAP channels are according to the Core Specification (3.A.2.1 v6.0) reserved
+ *        for the stack, but this allows the application to define their own, i. e. if there is
+ *        memory constraints and dynamic channels are not suitable. The application needs to make
```

Comment:
`s/i. e./i.e./` and "if there *are*" (since it's plural constraints)

Reply:
Fixed!